zap/src/main/java/org/parosproxy/paros/extension/history/PopupMenuExportMessage.java

```diff
@@ -57,9 +58,9 @@
 
     private static final long serialVersionUID = 1L;
 
-    private static final Logger log = LogManager.getLogger(PopupMenuExportMessage.class);
+    private static final Logger LOG = LogManager.getLogger(PopupMenuExportMessage.class);
 
-    private static final String CRLF = "\r\n";
+    private static final String EOL = System.lineSeparator();
```

Comment:
IMO it's better to use the same line endings always for consistency, I'd suggest changing the other export instead.

Reply:
You always use `\n`?

Reply:
I set it that way, lemme know if it was the wrong interpretation.

Reply:
Sounds good, worth noting that it will have mixed line endings (HTTP header/body separators) but any decent text editor should cope with that.